phabricator-user.rst

```diff
@@ -290,7 +284,7 @@ disabled in a Phabricator installation.  However, at the moment
 Mozilla has no defined engineering processes for post-commit review of
 Firefox and related code, so we do not recommend its use, at least
 until such time as a process is deemed necessary and implemented.
-Audit may, of course, be useful to projects hosted on the Mozilla
+An audit may, of course, be useful to projects hosted on the Mozilla
```

Comment:
"Audit" is the name of a Phabricator app, so it's being used correctly here; drop this change.